Dockerfile

```diff
@@ -6,8 +6,6 @@ ENV FOP_HOME=/usr/share/fop-2.1 \
 
 COPY --from=composer /usr/bin/composer /usr/bin/composer
```

Comment:
I think it would be nice to move this layer too, just in case there are changes in the composer executable.